src/components/security-code-error/security-code-error-controller.ts

```diff
@@ -18,6 +18,13 @@ export function securityCodeInvalidGet(req: Request, res: Response): void {
       Date.now() + getCodeEnteredWrongBlockDurationInMinutes() * 60000
     ).toUTCString();
   }
+
+  if (req.query.actionType === SecurityCodeErrorType.EmailMaxRetries) {
+    req.session.user.codeRequestLock = new Date(
```

Comment:
How comes you use the codeRequestLock here? I think it should be the`wrongCodeEnteredLock` but you will need to check that using this doesn't impact other parts of the journey such as when a user logs in and wants to enter their MFA code